app/Http/Controllers/LegacyMatchesController.php

```diff
@@ -36,10 +36,12 @@ public function __construct()
      * matches       | [Match](#match)[]             | |
      * params.limit  | integer                       | |
      * params.sort   | string                        | |
+     * params.active | boolean                       | |
      *
      * @usesCursor
      * @queryParam limit integer Maximum number of matches (50 default, 1 minimum, 50 maximum). No-example
      * @queryParam sort string `id_desc` for newest first; `id_asc` for oldest first. Defaults to `id_desc`. No-example
+     * @queryParam active boolean `true` for active matches only; `false` for inactive matches; Defaults to `null` returning both. No-example
```

Comment:
while passing string `null` works, it's not the correct value for the parameter. It should be either blank or not specified.